src/Clock/FixedClock.php

```diff
@@ -13,14 +13,11 @@
  */
 final class FixedClock implements Clock
 {
-    private Instant $instant;
-
     /**
      * @param Instant $instant The time to set the clock at.
      */
-    public function __construct(Instant $instant)
+    public function __construct(private Instant $instant)
```

Comment:
Shouldn't `StandaloneLinePromotedPropertyFixer` fix this?

Reply:
It only make them on their own lines when there are more than one. I've checked if there's an option for this behavior, but there's none.

Would you like me to change that by hand ?